README.md

```diff
@@ -363,6 +363,15 @@ interpreter, pass it on the command line at the time of build:
 PERL=/usr/local/my-fancy-perl pgenv build 10.5
 ```
 
+At the end of a `build` (or a `rebuild`) phase, `pgenv` creates a configuration
+file for the specific PostgreSQL version. If the file already exists, due to
+a prior `build` or `rebuild` action, the file will be automatically overwritten.
+In order to avoid the overwriting of the configuration file, it is possible to
+set the environment variable `PGENV_WRITE_CONFIGURATION_FILE_AUTOMATICALLY` to
+a false value (either `0` or `NO`). If the variable is not set at all,
+or it is set to a true value (e.g., `1`, `YES`, etc.)
+the configuration file will be overwritten.
```

Comment:
Isn't it the case that if `PGENV_WRITE_CONFIGURATION_FILE_AUTOMATICALLY` is false that the file won't be written at all, whether it exists or not?